components/profile_popover/profile_popover.jsx

```diff
@@ -188,6 +192,9 @@ class ProfilePopover extends React.PureComponent {
         delete popoverProps.actions;
         delete popoverProps.isTeamAdmin;
         delete popoverProps.isChannelAdmin;
+        delete popoverProps.intl;
+
+        const {formatMessage} = this.context.intl;
```

Comment:
I think we can use `this.props.intl` instead of accessing the context directly here.

Reply:
@hmhealey updated.